tests/linearGaussian_fmpe_test.py

```diff
@@ -287,7 +287,7 @@ def simulator(theta):
     # Evaluate the conditional density be drawing samples and smoothing with a Gaussian
     # kde.
     potential_fn, theta_transform = posterior_estimator_based_potential(
-        posterior_estimator, prior=prior
+        posterior_estimator, prior=prior, x_o=None
```

Comment:
Has `x_o` recently made a required positional argument?
I was just wondering as the previous code would have just raised an error.

Reply:
It has always been a required argument. And yes, the test failed because the error got raised. See #1276 for how the bug happened.